.github/workflows/continuous-integration.yml

```diff
@@ -28,15 +28,19 @@ jobs:
           - os: windows-latest
             java: 8
             epVersion: 2.4.0
+          - os: ubuntu-latest
+            java: 17
+            epVersion: 2.10.0
       fail-fast: false
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out NullAway sources
         uses: actions/checkout@v2
       - name: 'Set up JDK ${{ matrix.java }}'
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
```

Comment:
Took the opportunity to use the latest `setup-java` action and specify the `zulu` distribution (same as what Error Prone uses, though it shouldn't matter)